openapi_to_fastapi/validator/core.py

```diff
@@ -0,0 +1,45 @@
+import abc
+import json
+from pathlib import Path
+from typing import Union
+
+
+class OpenApiValidationError(Exception):
+    pass
+
+
+class InvalidJSON(OpenApiValidationError):
+    pass
+
+
+class UnsupportedVersion(OpenApiValidationError):
+    pass
+
+
+class MissingParameter(OpenApiValidationError):
+    pass
+
+
+class BaseValidator:
+    def __init__(self, path: Union[str, Path]):
+        if isinstance(path, str):
+            self.path = Path(path)
+        else:
+            self.path = path
```

Comment:
Actually `Path(Path("foo.txt")) == Path("foo.txt")`, so you could maybe simplify this to just `self.path = Path(path)`. Same thing in `routes.py:54`